Core/CachedAssetsDictionary.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Common;
+using Lykke.Service.Assets.Client.Models;
+
+namespace Core
+{
+    public class CachedAssetsDictionary : CachedDataDictionary<string, Asset>
```

Comment:
Please wrap up all CachedDataDictionary usage in IAssetsHelper as it is done in Assets controller.